timeline/templates/timeline/side_timeline.html

```diff
@@ -6,8 +6,7 @@
 {% load projectroles_common_tags %}
 
 {% block title %}
-  {{ timeline_title }} {% if timeline_mode == 'project' %}for{% else %}in{% endif %}
-  {{ project.title }}
+  {{ timeline_title }} in {{ project.title }}
```

Comment:
The project title should be omitted here, as we never have a project context in a site view.